src/openlifu/plan/__init__.py

```diff
@@ -1,7 +1,9 @@
 from .protocol import Protocol
+from .run import Run
```

Comment:
I think we can open another issue specifically for this for all other classes, because there is a problem of circular import that will probably need sometime to fix